02Lesson/01Homework/homework1.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Scott Foster</title>
+
+  <link rel="stylesheet"
+    href="homework1.css"
+    type="text/css">
+
+   <link rel="stylesheet"
+      href="reset.css"
+      type="text/css">
+
+</head>
+
+<body>
+  <h1 class="header1"> Scott Fosters super awesome introduction site</h1>
+
+  <section class="buttonlink">
+    <a class="button" href="#about-me">About Me</a>
+    <a class="button" href="#portfolio">Portfolio</a>
+    <a class="button" href="#contact">Contact</a>
+  </section>
+
+  <section>
+    <img class="propic" src="DSCF5677.jpg">
+    <p class="pictype">This is the only pic I have of myself</p>
+
+  </section>
+
+  <section>
+    <h2 id="about-me">About Me</h2>
+      <p>I am learning how to code. I am fairly proficient in HTML and CSS I am really in  need of work on my Javascript skills. This will come in time I am sure but That is the next skill I want to develop.
```

Comment:
Indentation. After closing `</h2>` tag next element should be on the same level as `<h2>`. That's why i would recomment putting block elements on their own line.